src/models/util/LoadShow.ts

```diff
@@ -45,14 +45,17 @@ function IsCalChart3(buffer: ArrayBuffer): ParseCalChart | null {
  * @returns Returns either a new [Show] or will throw an error that can be
  * displayed to the user
  */
-export const loadShowFromBuffer = (buffer: ArrayBuffer): Show => {
+export const loadShowFromBuffer = (
+  defaultTitle: string,
```

Comment:
Is `defaultTitle` added because CalChart3 shows don't have titles in their data?

Reply:
yep, for the most part that's what we see.

Reply:
I feel like the name `defaultTitle` is a little confusing, could we do something like `fileName` instead? For me, it was confusing where `defaultTitle` comes from... Is it a constant? Does it come from the user? Whereas with `fileName` we know where it originated from

Reply:
Great idea, will do.

Reply:
I'll have fileName be part of the InitialState, but then load will have a default title that will be string with the file name parsed off.